translator/src/test/java/com/google/devtools/j2objc/kotlin/EnumsTest.java

```diff
@@ -59,4 +61,32 @@ public void testSimpleEnumOrdinal() throws IOException {
 
         assertTranslation(translation, "return [((CommonSimpleEnum *) nil_chk(CommonSimpleEnum.enumvalue1)) ordinal];");
     }
+
+    @Test
+    public void testSimpleEnumSwitchCase() throws IOException {
+
+        String className = SimpleEnumSwitchCase.class.getSimpleName();
+        String translation = translateJavaSourceFileForKotlinTest(className, testPackage, ".m");
+
+        assertTranslation(translation, "switch ([testEnum ordinal]) {\n" +
+                "    case 0: // ENUMVALUE1\n" +
```

Comment:
Can't we do better?

Reply:
Maybe, but since this will only be seen in transpiled code. We figures it would be good enough so a human can understand pretty easily.

Reply:
As long as we revisit it later, readability is really important for stack traces when the app crashes.

Reply:
i would not count on use revisiting this if it works good enough. Since the end goal is not using j2objc anymore. The issue we had was the objc compiler only allows constants as case in a switch case. The objc outputed by Kotlin does not provide constants for enum ordinals.

Reply:
The end goal is to end up not using j2objc anymore (long shot i know) but some solution even if they are temporary, will stay in place during the migration process.